packages/mdl-button/_variables.scss

```diff
@@ -0,0 +1,20 @@
+/**
+ * Copyright 2016 Google Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+:root {
+  --mdl-primary-color: rgb(63, 81, 181);
```

Comment:
Rather than calling this `--mdl-primary-color`, `--mdl-accent-color`, etc. we should probably make sure that theme variables which pertain to components are scoped to those components. Also, we need to make sure to provide scss variables to rely on as fallbacks.

```scss
$mdl-button-text-color: color(black opacity(87%)) !default;
$mdl-button-raised-bg-color: white !default;

:root {
  --mdl-button-text-color: $mdl-button-text-color;
  --mdl-button-raised-bg-color: $mdl-button-raised-bg-color;
}
```

Then we can style based on those variables.

Primary/secondary/warn buttons are technically not in the spec. They can be easily implemented via either sass or custom variables:

```scss
.mdl-button.my-primary-button {
  --mdl-button-text-color: rgb(63, 81, 181);
  --mdl-button-raised-bg-color: rgb(63, 81, 181);
}
```